server/models/users.model.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require("mongoose");
+const { Schema } = mongoose;
+const passportLocalMongoose = require("passport-local-mongoose");
+const { Cart } = require("./cart.model");
+const { WishList } = require("./wishlist.model");
+const userSchema = new Schema({
+  firstname: String,
+  lastname: String,
+  email: String,
+  username: String,
+  email_id: String,
+  cartId: { type: Schema.Types.ObjectId, ref: Cart },
```

Comment:
you have email and email_id. what's the difference between the two? is this intentional?

Reply:
thanks for pointing it out